packages/block-library/src/query-pagination-previous/index.php

```diff
@@ -19,8 +19,18 @@
 function render_block_core_query_pagination_previous( $attributes, $content, $block ) {
 	$page_key            = isset( $block->context['queryId'] ) ? 'query-' . $block->context['queryId'] . '-page' : 'query-page';
 	$enhanced_pagination = isset( $block->context['enhancedPagination'] ) && $block->context['enhancedPagination'];
+	$max_page            = isset( $block->context['query']['pages'] ) ? (int) $block->context['query']['pages'] : 0;
 	$page                = empty( $_GET[ $page_key ] ) ? 1 : (int) $_GET[ $page_key ];
 
+	$total_pages = $max_page;
+	if ( ! isset( $block->context['query']['inherit'] ) || ! $block->context['query']['inherit'] ) {
```

Comment:
Can we move this whole block below here:https://github.com/WordPress/gutenberg/pull/67772/files#diff-f39b437ed48cda203e2489cc1acb59ed960f12669515152c85d2f6b630c81fdaR58?

Reply:
Hi @ntsekouras, Thanks for the review. I rebased the PR, and all the succeeding commits were pushed to it. That resulted in requesting reviews from many people. I don't think if that was expected out from this [comment](https://github.com/WordPress/gutenberg/pull/67772#pullrequestreview-2508636147), so I've created a new PR here: https://github.com/WordPress/gutenberg/pull/68070

I'll be proceeding to close this PR in contrast to the newly created PR.